dhis2/src/main/java/org/motechproject/dhis2/event/EventHandler.java

```diff
@@ -43,6 +44,14 @@
     @Autowired
     private DataElementService dataElementService;
 
+    public EventHandler (DhisWebService webService,
```

Comment:
We do not need two constructors, this bean should only get constructed with all dependencies. Since autowiring is done at the field level, this class doesn't need constructors at all. So for clarity sake just delete them both.